### ISSUE TITLE
`High Stride` TCon modifier (two pistons) not working

### ISSUE THREAD

**Issue by Krutoy242** [Krutoy242/Enigmatica2Expert-Extended#345]
Putting two pistons on Constructs' Armory boots does not grand 1-block step up.

![image](https://github.com/Krutoy242/Enigmatica2Expert-Extended/assets/936618/cee141bf-f464-4529-b49e-5e15717483a2)

**Modpack version** `v1.33.0`

Developer note:
Removing this mods does not help: `randompatches`, `UniversalTweaks`, `randomtweaks`, `RandomTweaker`

**Comment by LanceLander** [comment]
Don't know if it's related: this issue seems to extend to the Thaumcraft Belt of the Traveler, but NOT Boots of the Traveler. 

As well, Boots of the Traveler no longer give extra jump height and will overwrite the jump height increase if wearing them with Belt of the Traveler.

Modpack version v1.35.0

**Comment by Invadermonky** [comment]
Jump height effects being broken is a result of `B:"Controllable Jump Boost"=true` in `potioncore.cfg`. Setting it to false fixes these issues.

**Comment by Krutoy242** [comment]
> B:"Controllable Jump Boost"=true

Sadly, but setting `B:"Controllable Jump Boost"=false` does not fixing "High Stride" issue and player still cant step up on a full block.

**Comment by Invadermonky** [comment]
The config option was sort of a response to:
> As well, Boots of the Traveler no longer give extra jump height and will overwrite the jump height increase if wearing them with Belt of the Traveler.

I found that bug a long time ago (after a painstaking binary search) and have been trying to spread the word about it since a lot of packs suffer from the issue.

Setting the config option to false fixes Boots of the Traveler jump boost, Living Armor jump boost, and a few others I can't remember off the top of my head.

**Comment by Krutoy242** [comment]
Oh, thank you then! I will keep this option disabled.

But High Stride still unsolved...

**Comment by DDriggs00** [comment]
Is it possible there's an unlisted keybind to toggle it?

**Comment by Krutoy242** [comment]
No, because some other methods of stepup not working too.
Cant recall which exactly, maybe enchantment or modded bauble.

**Comment by DDriggs00** [comment]
The Astral Sorcery and Ender IO Armor methods are also affected by the same bug.

**Comment by astralstriker** [comment]
Speedy Modifier isn't working either. I see no change in my movement speed whatsoever.

**Comment by Krutoy242** [comment]
> Speedy Modifier isn't working either. I see no change in my movement speed whatsoever.

Just tested and `Speedy Modifier` working as it should - it give speed boost, up to 60%. Seems like you applying it wrong, or have too low level. Please follow E2EE Discord group if you need help to figure it out.

**Comment by astralstriker** [comment]
Yes, I have only 19 dust put in. It shows a small buff to ms. It taking multiple modifier slots had me wondering. Thanks for the clarification.

**Comment by DDriggs00** [comment]
Has anyone done a binary mod removal to find the mods causing this yet?

**Comment by Krutoy242** [comment]
> Has anyone done a binary mod removal to find the mods causing this yet?

I did and have no luck - seems like error depends not on 2 mods but on 3 mods, which binary search barely can help.

**Comment by DDriggs00** [comment]
I've tracked the problem down to ArtemisLib, an optional depencency of the Minor Integrations and Additions mod.

**Comment by Krutoy242** [comment]
> I've tracked the problem down to ArtemisLib, an optional depencency of the Minor Integrations and Additions mod.

So, does removing ArtemisLib alone fix the issue? ArtemisLib was added only to enable size change feature.

**Comment by Anthonity** [comment]
Removing ArtemisLib fixed the problem apparently

**Comment by Krutoy242** [comment]
Oh.... Thats good catch! With this information we could fix the issue faster.

Could you play with `ArtemisLib` options to fix the issue? Maybe there some configuration that can fix problem without removing mod?

**Comment by Anthonity** [comment]
> Oh.... Thats good catch! With this information we could fix the issue faster.
> 
> Could you play with `ArtemisLib` options to fix the issue? Maybe there some configuration that can fix problem without removing mod?

There doesn't seem to be any configs for it unfortunately

**Comment by Anthonity** [comment]
Also, I forgot to mention tested an instance with just ArtemisLib, MIA and some of the affected mods, and the step assist works just fine. Weird.

<img width="689" height="431" alt="Image" src="https://github.com/user-attachments/assets/e07307fa-c9ec-421f-8b52-e1ce017f9a8c" />

**Comment by Krutoy242** [comment]
This what i'm talking about. Its probably not just Artemis Lib but also some other mod.

**Comment by melik3567** [comment]
if you wanna step assist buff i found one solution
cyclic apple (i dont know the item name) it work 
I think it works because it turns our codes into horses and all other buffs are based on human code.
